worker/src/env.rs

```diff
@@ -1,6 +1,6 @@
-use crate::error::Error;
 #[cfg(feature = "queue")]
 use crate::Queue;
+use crate::{d1::D1Database, error::Error};
```

Comment:
While D1 is still in alpha we should gate it behind a feature flag like we do for queues.

Reply:
Good idea, will implement the feature flag.